.changeset/hip-crabs-vanish.md

```diff
@@ -0,0 +1,7 @@
+---
+"@khanacademy/perseus": major
+"@khanacademy/perseus-core": major
+"@khanacademy/perseus-score": major
```

Comment:
Is this too extreme? It's a breaking change but for a use that never made sense.

Reply:
If someone had logic that called `splitPerseusItem` without registering (and for some reason thought that was a good thing), this would change the behavior of their code.

It's not _technically_ a bug fix and it does change behavior.